docs/studio.mdx

```diff
@@ -109,7 +109,7 @@ As a [Pro user](https://stately.ai/pricing), you can create and join teams in th
 
 <p>
   <ThemedImage
-    alt="Stately Studio Team page for the Voyager team, showing Captain Janeway with the owner and admin role, Commander Chakotay with an admin role, Lieutenant Tuvok, Lieutenant Torees, and Lieutenant Paris with Editor roles, and Ensign Kim with a Viewer role."
+    alt="Stately Studio Team page for the Voyager team, showing Captain Janeway with the owner and admin role, Commander Chakotay with an admin role, Lieutenant Tuvok, Lieutenant Torees, and Lieutenant Paris with editor roles, and Ensign Kim with a Viewer role."
```

Comment:
Sometime we'll figure out whether these team roles are capitalized or not so we can be more consistent (separate task).

Reply:
But I think the find/replace wasn't meant to lowercase this `editor`.